host/sgx/sgx_enclave_common_wrapper.h

```diff
@@ -33,6 +33,10 @@
     (1 << ENCLAVE_CREATE_EX_EL_RANGE_BIT_IDX) // Reserve Bit 0 for the el_range
 #endif
 
+#define OE_SE_PAGE_SIZE 0x1000
+#define OE_SE_PAGE_SIZE_MASK 0xFFF
+#define OE_ADDRESS_ZERO 0x0
```

Comment:
if these are SGX specific, can we add SGX into the define names?

Reply:
implemented.